drivers/rtc/rtc_mc146818.c

```diff
@@ -51,7 +51,6 @@
 #define RTC_UIP		RTC_REG_A
 #define RTC_DATA	RTC_REG_B
 #define RTC_FLAG	RTC_REG_C
-#define RTC_ALARM_MDAY	RTC_REG_D
```

Comment:
How much of this file actually needed to be changed in order for the fix to work?

At a glance, it seems that most of the changes here are cosmetic (formatting, name changes, ...). Is it possible to reduce the change to only what is required for the fix?

Reply:
These changes are the bare minimum required to get the driver to an acceptable state. The driver still needs a once over to clean it up, which I agree is not within the scope of this PR